src/DfE.CoreLibs.Security/Interfaces/ICustomRequestChecker.cs

```diff
@@ -20,5 +21,10 @@ public interface ICustomRequestChecker
         /// otherwise, <c>false</c>.
         /// </returns>
         bool IsValidRequest(HttpContext httpContext);
+
+        /// <summary>
+        /// Gets the operator type that defines how multiple
+        /// </summary>
+        OperatorType Operator { get; }
     }
```

Comment:
Not sure if this approach works, you need to create an OperatorGroup class as I mentioned yesterday, then let that receive a collection of checker type names and the operator, that would give you flexibility to choose what operator to be used in between what checker types